Mathlib/Topology/Homeomorph.lean

```diff
@@ -266,6 +266,19 @@ theorem isCompact_preimage {s : Set β} (h : α ≃ₜ β) : IsCompact (h ⁻¹'
   rw [← image_symm]; exact h.symm.isCompact_image
 #align homeomorph.is_compact_preimage Homeomorph.isCompact_preimage
 
+/-- If `h : α → β` is a homeomorphism, `s` is σ-compact iff `h(s)` is. -/
+theorem isSigmaCompact_image {s : Set α} (h : α ≃ₜ β) :
```

Comment:
This looks like a good `simp` lemma to me, unless the linter knows better.

Is this (or the `preimage` version) true under the weaker assumption `Embedding f`?

Reply:
Indeed it is --- this is shown in `Embedding.isSigmaCompact_iff_isSigmaCompact_image` in `SubsetProperties.lean`.

I have pushed a change tagging this simp; let's see what CI says.

Reply:
Can you use that lemma to prove this one?